### PR TITLE
Use StdBaroPress in Component Load Summary

### LINKED ISSUE #9728: Table reports use standard pressure instead of pressure at altitude
Issue overview
--------------
Code review shows use of a hard-wired ambient pressure (STP) for zone RH and wet-bulb temperature (4 occurences in CollectPeakZoneConditions).

            compLoad.outsideWetBulb = PsyTwbFnTdbWPb(state,
                                                 CalcFinalZoneSizing(zoneIndex).CoolOutTempSeq(timeOfMax),
                                                 CalcFinalZoneSizing(zoneIndex).CoolOutHumRatSeq(timeOfMax),
                                                 101325.0);

            compLoad.zoneRelHum = PsyRhFnTdbWPb(state,
                                                CalcFinalZoneSizing(zoneIndex).CoolZoneTempSeq(timeOfMax),
                                                CalcFinalZoneSizing(zoneIndex).CoolZoneHumRatSeq(timeOfMax),
                                                101325.0);

This should be StdBaroPress to be consistent with simulation results.

image

Issue from Trane North America.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added ([5ZoneAirCooled-Modified.idf.txt](https://github.com/NREL/EnergyPlus/files/9939897/5ZoneAirCooled-Modified.idf.txt)  with Denver Intl weather.)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
 - Fixes #9728
 - Component Load Summary reports now use Standard Barometric Pressure at altitude to calculate moist air properties of outdoor wet-bulb temperature and zone relative humidity.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [x] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [x] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [x] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [x] Perform a Code Review on GitHub
 - [x] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [x] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] ~If feature, test running new feature, try creative ways to break it~
 - [x] CI status: all green or justified
 - [x] Check that performance is not impacted (CI Linux results include performance check)
 - [x] Run Unit Test(s) locally
 - [ ] ~Check any new function arguments for performance impacts~
 - [ ] ~Verify IDF naming conventions and styles, memos and notes and defaults~
 - [ ] ~If new idf included, locally check the err file and other outputs~

### CODE REVIEW

File: src/EnergyPlus/OutputReportTabular.cc [hunk 1]
```diff
@@ -16302,50 +16299,49 @@ void CollectPeakZoneConditions(
...
-                compLoad.outsideWetBulb = PsyTwbFnTdbWPb(state,
-                                                         CalcFinalZoneSizing(zoneIndex).CoolOutTempSeq(timeOfMax),
-                                                         CalcFinalZoneSizing(zoneIndex).CoolOutHumRatSeq(timeOfMax),
-                                                         101325.0);
+            // use standard air pressure because air pressure is not tracked with sizing data
+            if (calcFinalZoneSizing.CoolOutHumRatSeq(timeOfMax) < 1.0 && calcFinalZoneSizing.CoolOutHumRatSeq(timeOfMax) > 0.0) {
+                compLoad.outsideWetBulb = Psychrometrics::PsyTwbFnTdbWPb(state,
+                                                                         calcFinalZoneSizing.CoolOutTempSeq(timeOfMax),
+                                                                         calcFinalZoneSizing.CoolOutHumRatSeq(timeOfMax),
+                                                                         state.dataEnvrn->StdBaroPress);
```
Comment (rraustad) [inline src/EnergyPlus/OutputReportTabular.cc:16310]: Issue correction to use StdBaroPress for moist air property calculations (4 places).

**Comment by rraustad** [comment]
Example of correction to moist air property calculation is shown in issue discussion #9728.

**Comment by rraustad** [comment]
Diffs as expected in Component Load Summary tables. Interesting that 2 err files show this warning is no longer reported.

        ************* All Return Air Paths passed integrity testing
        ************* No node connection errors were found.
        ************* Beginning Simulation
    -   ** Warning ** Calculated Relative Humidity out of range (PsyRhFnTdbWPb) 
    -   **   ~~~   **  Routine=Unknown, Environment=RUNPERIOD 1, at Simulation time=07/21 24:00 - 24:15
    -   **   ~~~   **  Dry-Bulb= 29.98 Humidity Ratio= 2.783E-002 Calculated Relative Humidity [%]= 102.28
    -   **   ~~~   ** Relative Humidity being reset to 100.0%
        ************* Simulation Error Summary *************
    -   *************
    -   ************* ===== Recurring Error Summary =====
    -   ************* The following recurring error messages occurred.
    -   *************
    -   *************  ** Warning ** Calculated Relative Humidity out of range (PsyRhFnTdbWPb)
    -   *************  **   ~~~   **   This error occurred 1 total times;
    -   *************  **   ~~~   **   during Warmup 0 times;
    -   *************  **   ~~~   **   during Sizing 0 times.
    -   *************  **   ~~~   **   Max=102.283395 %  Min=102.283395 %
    -   *************
        ************* EnergyPlus Warmup Error Summary. During Warmup: 0 Warning; 0 Severe Errors.
        ************* EnergyPlus Sizing Error Summary. During Sizing: 0 Warning; 0 Severe Errors.

**Review by mjwitte — APPROVED** [review]
Code changes look good. The RH reported in the zone component loads summary now matches the same value reported in Coil Sizing Details (for heating).
Since it's been a while, I've updated the branch and will let CI chew on it one more time.
Thanks @rraustad

**Pull request merged**